Sources/MapboxCoreNavigation/CoreNavigationNavigator.swift

```diff
@@ -45,6 +45,8 @@ class Navigator {
         return RoadObjectsStore(try! navigator.roadObjectStore())
     }()
     
+    static var accessToken: String? = nil
+    
     /**
      Provides a new or an existing `MapboxCoreNavigation.Navigator` instance. Upon first initialization will trigger creation of `MapboxNavigationNative.Navigator` and `HistoryRecorderHandle` instances,
      satisfying provided configuration (`tilesVersion` and `tilesURL`).
```

Comment:
Add a `- precondition:` mentioning that the property above should be set before getting the shared navigator for the first time.